test/blocks/merch-card/merch-card.test.js

```diff
@@ -93,6 +99,11 @@ describe('Merch Card', () => {
       expect(buttons[0].textContent).to.be.equal('Learn More');
       expect(buttons[1].textContent).to.be.equal('Save now');
       expect(secureWrapper).to.be.exist;
+      expect(list).to.be.exist;
+      expect(list.classList.contains('consonant-PlansCard-list')).to.be.true;
+      expect(listItems.length).to.be.equal(2);
+      expect(listItems[0].textContent).to.be.equal('Maecenas porttitor congue massa');
+      expect(listItems[1].textContent).to.be.equal('Nunc viverra imperdiet enim.');
```

Comment:
same here, probably can remove the `be` and int he matchers on lines 55, 58-61

Reply:
fixed